docs/qa/test_engineering

```diff
@@ -0,0 +1,44 @@
+Normandy Test Engineering
```

Comment:
You'll need to name this file `test_engineering.rst`.

Reply:
done